packages/react-hooks/src/preferences/types.ts

```diff
@@ -4,6 +4,7 @@ export interface PreferenceState {
   preferences?: IPreferenceTemplate[];
   recipientPreferences?: IRecipientPreference[];
   preferenceSections?: Array<any>;
+  preferencePage?: any;
```

Comment:
do we need preferenceSections and preference page at the same level?

Reply:
for now, I'll create another ticket to move deprecate preferenceSections.